### PR TITLE
fixes log timestamp

### PR DESCRIPTION
**Pull request by vbnrh**
Configures the zap logger to display timestamp in ISO format
Previous output example:
```
{"level":"info","ts":1569828960.6597447,"logger":"cmd","msg":"Go Version: go1.11.13"}
{"level":"info","ts":1569828960.6597664,"logger":"cmd","msg":"Go OS/Arch: linux/amd64"}
{"level":"info","ts":1569828960.6597693,"logger":"cmd","msg":"operator-sdk Version: v0.10.0"}
{"level":"info","ts":1569828960.6600604,"logger":"leader","msg":"Trying to become the leader."}
```
Output example:
```
{"level":"info","ts":"2019-12-05T09:42:14.107Z","logger":"cmd","msg":"Go Version: go1.13.4"}
{"level":"info","ts":"2019-12-05T09:42:14.108Z","logger":"cmd","msg":"Go OS/Arch: linux/amd64"}
{"level":"info","ts":"2019-12-05T09:42:14.108Z","logger":"cmd","msg":"operator-sdk Version: v0.10.0"}
{"level":"info","ts":"2019-12-05T09:42:14.108Z","logger":"leader","msg":"Trying to become the leader."}
{"level":"info","ts":"2019-12-05T09:42:16.286Z","logger":"leader","msg":"No pre-existing lock was found."}
{"level":"info","ts":"2019-12-05T09:42:16.296Z","logger":"leader","msg":"Became the leader."}
```

### CODE REVIEW

**Comment by kshlm** [comment]
/hold

This change needs to be done on master first.

**Comment by kshlm** [comment]
@vbnrh Can you also provide an example how the log timestamp will be after this change is merged?

File: cmd/manager/main.go [hunk 1]
```diff
@@ -148,3 +153,36 @@ func main() {
...
+	logger := zapr.NewLogger(getZapLogger(os.Stderr, false))
+	logf.SetLogger(logger)
+}
+
+func getZapLogger(destWriter io.Writer, development bool, opts ...zap.Option) *zap.Logger {
+	sink := zapcore.AddSync(destWriter)
+
+	var enc zapcore.Encoder
+	var lvl zap.AtomicLevel
+	if development {
```
Comment (jarrpa) [inline cmd/manager/main.go]: Why are you including this stanza if it is never used?

Comment (vbnrh) [reply cmd/manager/main.go]: @jarrpa It is to enable the debug logs by passing a true in the getZapLogger func

Comment (vbnrh) [reply cmd/manager/main.go]: @jarrpa removed BZ from the title

**Comment by vbnrh** [comment]
> @vbnrh Can you also provide an example how the log timestamp will be after this change is merged?

Added in description

File: cmd/manager/main.go (hunk 1 above)
Comment (kshlm) [reply cmd/manager/main.go]: > @jarrpa It is to enable the debug logs by passing a true in the getZapLogger func

But we are not doing that anywhere. Do you plan to introduce a flag or better a ENVVAR to enable a debug logging mode?

File: pkg/apis/ocs/v1/zz_generated.openapi.go [hunk 2]
```diff
@@ -11,14 +11,14 @@ import (
...
-		"github.com/openshift/ocs-operator/pkg/apis/ocs/v1.OCSInitializationSpec":              schema_pkg_apis_ocs_v1_OCSInitializationSpec(ref),
-		"github.com/openshift/ocs-operator/pkg/apis/ocs/v1.OCSInitializationStatus":            schema_pkg_apis_ocs_v1_OCSInitializationStatus(ref),
-		"github.com/openshift/ocs-operator/pkg/apis/ocs/v1.StorageCluster":                     schema_pkg_apis_ocs_v1_StorageCluster(ref),
-		"github.com/openshift/ocs-operator/pkg/apis/ocs/v1.StorageClusterInitialization":       schema_pkg_apis_ocs_v1_StorageClusterInitialization(ref),
-		"github.com/openshift/ocs-operator/pkg/apis/ocs/v1.StorageClusterInitializationSpec":   schema_pkg_apis_ocs_v1_StorageClusterInitializationSpec(ref),
-		"github.com/openshift/ocs-operator/pkg/apis/ocs/v1.StorageClusterInitializationStatus": schema_pkg_apis_ocs_v1_StorageClusterInitializationStatus(ref),
-		"github.com/openshift/ocs-operator/pkg/apis/ocs/v1.StorageClusterStatus":               schema_pkg_apis_ocs_v1_StorageClusterStatus(ref),
+		"github.com/ocs-operator/ocs-operator/pkg/apis/ocs/v1.OCSInitialization":                  schema_pkg_apis_ocs_v1_OCSInitialization(ref),
```
Comment (kshlm) [inline pkg/apis/ocs/v1/zz_generated.openapi.go]: This doesn't seem right. How did you end up changing the package path here? Please revert it.

**Comment by kshlm** [comment]
/hold cancel

PR is on master now.

File: pkg/apis/ocs/v1/zz_generated.openapi.go (hunk 2 above)
Comment (kshlm) [reply pkg/apis/ocs/v1/zz_generated.openapi.go]: Seems like you've cloned the ocs-operator repo under a path different to `$GOPATH/src/github.com/openshift/ocs-operator`. This is the generate code is generating different package name.

Please move the ocs-operator repo to `$GOPATH/src/github.com/openshift/ocs-operator` and re-run `make update-generated`.

Comment (vbnrh) [reply pkg/apis/ocs/v1/zz_generated.openapi.go]: Changes completed.. repo has been moved to the correct path

File: cmd/manager/main.go (hunk 1 above)
Comment (vbnrh) [reply cmd/manager/main.go]: @kshlm Will work on this feature in a follow up PR.

**Comment by vbnrh** [comment]
> Looks good. But the way the PR is split into commits isn't good. Please rearrange the commits into 2 commits.
> 
> 1. First commit just introducing the `initLogger` change.
> 2. A second commit with the vendor changes, including the update to go.mod and go.sum.

i have merged the first two commits for initLogger change which already dependency files included. 
The 2nd commit corrects the path issue with the generated code.

**Pull request merged**